client/verta/docs/tutorials/deployment/endpoint_config.rst

```diff
@@ -0,0 +1,89 @@
+Endpoint configuration
+======================
+
+Through an endpoint update, you can configure properties of the deployment including its
+autoscaling behavior and compute resources.
+
+Using the client
+----------------
+
+:meth:`Endpoint.update() <verta._deployment.endpoint.Endpoint.update>` provides parameters for
+configuring the resources and behavior of a deployed model. These settings can be used alongside
+any update strategy.
+
+.. code-block:: python
+
+    from verta.deployment.update import DirectUpdateStrategy
+
+    endpoint.update(
+        model_version, DirectUpdateStrategy(),
+        autoscaling=autoscaling,
+        resources=resources,
+        env_vars=env_vars,
+    )
+
+Autoscaling
+^^^^^^^^^^^
+
+``autoscaling`` takes an :class:`~verta.deployment.autoscaling._autoscaling.Autoscaling` object,
+which itself is used to establish upper and lower bounds for the number of replicas running the
+model. Autoscaling must also have at least one metric associated with it, which sets a threshold
+for triggering a scale-up.
+
+.. code-block:: python
+
+    from verta.deployment.autoscaling import Autoscaling
+    from verta.deployment.autoscaling.metrics import CpuUtilizationTarget
+
+    autoscaling = Autoscaling(max_replicas=4, min_scale=0.5)
+    autoscaling.add_metric(CpuUtilizationTarget(0.75))
+
+Here, CPU utilization exceeding 75% will lead to more replicas being created. For the full list of
+available metrics, see the :ref:`autoscaling` API documentation.
+
+Resources
+^^^^^^^^^
+
+``resources`` specifies the computational resources that will be available to the model when it is
+deployed.
+
+.. code-block:: python
+
+    from verta.deployment.resources import (CpuMillis, Memory)
+
+    resources = [
+        CpuMillis(250),
+        Memory("512Mi"),
+    ]
+
+In this example, each replica will be provided a fourth of a CPU core and 512 Mi of RAM. For the full
+list of available resources, see the :ref:`resources` API documentation.
+
+Environment variables
+^^^^^^^^^^^^^^^^^^^^^
+
+``env_vars`` takes a dictionary of string environment variable names to string values, and will be
+made available to the model when it is deployed.
+
+.. code-block:: python
+
+    env_vars = {'CUDA_​VISIBLE_​DEVICES': "1,2"}
```

Comment:
I would advise against providing examples with `CUDA_​VISIBLE_​DEVICES` as that has a very strong implicit meaning. I would recommend picking an example with less loaded meaning.

Reply:
@conradoverta 
Could you recommend another example? This was the only env var I could think of that has some relevant to ML.

Reply:
`LOG_LEVEL=debug`?